src/app/components/file-uploader/file-uploader.component.ts

```diff
@@ -0,0 +1,163 @@
+import { Component, ElementRef, Input, ViewChild } from '@angular/core';
+import { HttpEventType } from '@angular/common/http';
+import { forkJoin, Observable, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { AlertConfig } from 'src/app/models/internal/alert-config';
+import { AlertType } from 'src/app/models/internal/alert-type';
+import { AlertService } from 'src/app/services/alert.service';
+import { FileUploaderService } from 'src/app/services/file-uploader.service';
+import { UploadFile } from 'src/app/models/domain/uploadFile';
+import { FileRetrieverService } from 'src/app/services/file-retriever.service';
+
+/**
+ * Component that will function as a form to upload files of any type
+ */
+@Component({
+  selector: 'app-file-uploader',
+  templateUrl: './file-uploader.component.html',
+  styleUrls: ['./file-uploader.component.scss']
+})
+export class FileUploaderComponent {
+
+  @ViewChild('fileDropRef') fileInput: ElementRef;
+  @Input() acceptMultiple: boolean;
+  @Input() acceptedTypes: Array<String>;
+
+  private maxFileSize = 2097152;
```

Comment:
Comment above to describe what this means. Is this in bytes? Also might be nice to show this to the user.